docs/docs/networks/picasso/governance.md

```diff
@@ -1,28 +1,73 @@
 
-## Governance Divisions
+The new OpenGov structure for Picasso will be implemented in two phases.
+
+Phase 1: Implement OpenGov with governance handled by two collectives: The Picasso Council and the Technical Committee.
+
+This phase lays the foundation for a more transparent and community-driven governance model while maintaining the speed and efficiency demonstrated by the recent launches and connections on Picasso, especially in preparation for our upcoming IBC connection to Ethereum.
+
+Phase 2: Release OpenGov to public PICA holders.
+
+The second phase will be implemented after the launch of the Ethereum ⬌ IBC connection which will be released in Q4 2023. Through OpenGov, the Picasso parachain will undergo a new era of decentralization, allowing PICA holders to actively participate in governance and a new structure of collectives.
+
+## OpenGov Tracks
+
+During Phase 1, Council members and the Technical Committee will be able to vote via a GOV token. The GOV token has no monetary value and held by the Picasso Council and Technical Committee members. Initially, there are two tracks, and four origins:
+
+- Root
+- Whitelisted Caller (Fast Track)
+
+Additional tracks are also planned to be added during Phase 1 such as General Admin, Treasury and more.
+
+The governance model during Phase 1 will introduce four separate roles, each serving a unique purpose:
+
+- Proposal Creation: Enabling the ability to create proposals, fostering innovation and community-driven development.
+- Voting: Allowing GOV holders to vote on proposals, ensuring that decisions reflect the collective will of the community.
+- Canceling Proposals/Slashing: Implementing mechanisms to cancel proposals and penalize those who voted maliciously..
+- Expedition: Introducing measures to speed up the voting process when needed, ensuring timely decision-making without compromising on democratic principles.
+
+These tracks and origins are designed to ensure a balanced and fair approach, aligning with our commitment to transparency, decentralization, and community engagement.
+
+## OpenGov Parameters
+
+The following two tables provide information about the voting and decision-making processes for the two different collectives and tracks (Whitelist Track and Root Track). They summarize the support thresholds required for various actions, the time it takes for those actions to pass when specific support percentages are met, and the confirmation and decision periods for each track.
+
+
+
+| Collective           | Support Threshold | Whitelist Time (Passing) | Root Time (Passing) |
+|-----------------------|-------------------|--------------------------|----------------------|
+| Technical Committee     | 10%               | 12 hours        | 5 days 14 hours      |
```

Comment:
assuming approval 100 percent referenda will pass after this time with this much support